server/auth/types/openid/routes.ts

```diff
@@ -100,6 +100,7 @@ export class OpenIdAuthRoutes {
                   validate: validateNextUrl,
                 })
               ),
+              redirectHash: schema.maybe(schema.string()),
```

Comment:
Why string and not boolean here?

Reply:
This is the hash in the url.

i.e. if url is `http://localhost:5601/app/security-dashboards-plugin#/getstarted`

then redirectHash is `#/getstarted`

Reply:
Changed to `boolean`